### ISSUE TITLE
Upload sourcemap for web demo

### ISSUE THREAD

**Issue by ueman** [getsentry/sentry-dart#430]
We should upload source maps for the web demo at https://getsentry.github.io/sentry-dart/#/

**Comment by bruno-garcia** [comment]
For reference, there's a 'run.sh' script that was uploading symbols:

https://github.com/getsentry/sentry-dart/blob/a1f7888666a7c1a1efc9453b008c23284c3e4207/flutter/example/run.sh#L39-L53

**Comment by ueman** [comment]
This seems to be blocked by https://github.com/getsentry/sentry-dart/issues/417

**Comment by marandaneto** [comment]
@ueman I think it works to some degree, do you remember whats the limitation @bruno-garcia ?

**Comment by bruno-garcia** [comment]
I can't recall what the limitations are. We might need to circle back with other folks about it.
We could push the sourcemaps though and see what we do from there.

**Comment by shripal17** [comment]
Hi,
I use this script for creating sentry releases whenever i deploy my web-app.
```bash
# $1 = version
# $2 = env (dev/prod)

echo "Doing sentry release for version $1 env $2"
sentry-cli releases new "$1"
sentry-cli releases set-commits "$1" --auto
sentry-cli releases files "$1" upload-sourcemaps . --ext dart --rewrite
sentry-cli releases files "$1" upload-sourcemaps ../lms_content --ext dart --rewrite --url-prefix '~/lms_content/'
sentry-cli releases files "$1" upload-sourcemaps ./build/web/ --ext map --ext js
sentry-cli releases finalize "$1"
sentry-cli releases deploys "$1" new -e $2
```

This used to work perfectly before we migrated to null-safety.
But after migrating to null-safety, sentry doesn't seem to process the uploaded sourcemaps and just shows JS errors.

Can you please help me to fix this?

**Comment by marandaneto** [comment]
@shripal17 I'm not sure if Dart/Flutter have changed their tooling, but we already have this issue https://github.com/getsentry/sentry-dart/issues/417 to track source maps.
can you check if you or the dart/flutter tooling have changed and what? thanks
on our end, nothing has changed really, just using the new feature of the language (null-safety)

**Comment by shripal17** [comment]
@marandaneto I am not sure how I would be able to check with dart/flutter changes.
But I just noticed, one of the errors was successfully able to trace through the sourcemaps.
I have no idea why this one worked and why don't other errors work properly.
I had changed nothing in the release script which would make this particular issue eligible for tracing.
Other errors from the same release fail to trace.

**Comment by shripal17** [comment]
![image](https://user-images.githubusercontent.com/16012350/123393284-90362400-d5bb-11eb-9c40-8e1f7768a7fc.png)

As you can see in this screenshot, the first seen issue (last in the list) here was able to perfectly trace through dart source. But subsequent errors just show `main.dart` and they just trace back to `main.dart.js`

Most probably the second last issue is the same as the last one.

**Comment by marandaneto** [comment]
@shripal17 please raise a new issue with all the info you have, this issue is unrelated to your issue.
